sources/install/package_ad.sh

```diff
@@ -140,6 +140,71 @@ function install_bloodhound() {
     add-to-list "bloodhound,https://github.com/BloodHoundAD/BloodHound,Active Directory security tool for reconnaissance and attacking AD environments."
 }
 
+function install_bloodhound-ce() {
+    # CODE-CHECK-WHITELIST=add-aliases,add-history
+    colorecho "Installing BloodHound-CE"
+
+    # Installing & Configuring the database
+    fapt postgresql postgresql-client
+    # only expose postgresql on localhost
+    sed -i 's/#listen_addresse/listen_addresse/' /etc/postgresql/15/main/postgresql.conf
+    service postgresql start
+    # avoid permissions issues when impersonating postgres
+    cd /tmp || exit
+    sudo -u postgres psql -c "CREATE USER bloodhound WITH PASSWORD 'exegol4thewin';"
+    sudo -u postgres psql -c "CREATE DATABASE bloodhound;"
+    sudo -u postgres psql -c "ALTER DATABASE bloodhound OWNER TO bloodhound;"
+    service postgresql stop
+
+    # Build BloodHound-CE
+    mkdir -p /opt/tools/BloodHound-CE/
+    git -C /opt/tools/BloodHound-CE/ clone --depth 1 https://github.com/SpecterOps/BloodHound.git src
+    cd /opt/tools/BloodHound-CE/src/packages/javascript/bh-shared-ui || exit
+    zsh -c "source ~/.zshrc && nvm install 18 && nvm use 18 && yarn install --immutable && yarn build"
+    cd /opt/tools/BloodHound-CE/src/ || exit
+    catch_and_retry VERSION=v999.999.999 CHECKOUT_HASH="" python3 ./packages/python/beagle/main.py build --verbose --ci
+
+    # Collectors
+    mkdir -p /opt/tools/BloodHound-CE/collectors/sharphound
+    mkdir -p /opt/tools/BloodHound-CE/collectors/azurehound
+    ## SharpHound
+    local SHARPHOUND_URL
+    SHARPHOUND_URL=$(curl --location --silent "https://api.github.com/repos/BloodHoundAD/SharpHound/releases/latest" | grep 'SharpHound-.*.zip' | grep -v 'debug' | grep -o 'https://[^"]*')
+    wget --directory-prefix /opt/tools/BloodHound-CE/collectors/sharphound/ "$SHARPHOUND_URL"
```

Comment:
Sharphound shouldn't be part of exegol resources ?
However, we can add a symlink to the exegol resources sharphound directory ?

Reply:
In theory, yes.
In practice, bloodhound-ce requires the ingestors to be in a specific directory and checks that when starting. We'll probably need to have ingestors installed in the image as well as the resources